corehq/apps/app_manager/util.py

```diff
@@ -146,11 +149,24 @@ def get_properties(self, case_type, already_visited=(),
             for app in self.get_other_case_sharing_apps_in_domain():
                 case_properties.update(
                     get_case_properties(
-                        app, [case_type], include_shared_properties=False
+                        app, [case_type], include_shared_properties=False, usercasetype=usercasetype
                     ).get(case_type, [])
                 )
 
-        return case_properties
+        # prefix user case properties with "user:".
+        prefix_user = lambda p: USERCASE_PREFIX + p if case_type == usercasetype else p
+
+        # .. note:: if the user case type has a parent case type, its
+        #           properties will be returned as `user:parent/property`
+        #
+        # .. note:: if this case type is not the user case type, but it has a
+        #           parent case type which is the user case type, then the
+        #           parent case type's properties will be returned as
+        #           `parent/user:property`. That's probably unlikely, and a
+        #           good thing that it is, because that looks like a mess, and
+        #           will likely cause problems.
```

Comment:
This may not be as unlikely as you think. I can quite easily foresee people creating child cases of the user case e.g. one case per exam.

I think its fine not to handle that now but is likely something we'll need to figure out.